src/main/java/JeM.java

```diff
@@ -0,0 +1,107 @@
+import java.util.Scanner;
+
+public class JeM {
+
+    public static void main(String[] args) {
+        displayWelcomeMessage();
+
+        Scanner scanner = new Scanner(System.in);
+        Storage storage = new Storage();
+
+        while (true) {
+            String line = scanner.nextLine().trim();
+
+            if (line.equalsIgnoreCase("bye")) {
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+            }
+
+            handleInput(line, storage);
+        }
+
+        System.out.println("____________________________________________________________");
```

Comment:
It will be better if you could extract this line of code to create a printLine() method, so you wouldn't have to copy and paste this line throughout the code.